cmd/candi/template_repository.go

```diff
@@ -20,6 +20,7 @@ func SetSharedRepository(deps dependency.Dependency) {
 	once.Do(func() {
 		{{if not .SQLDeps}}// {{end}}setSharedRepoSQL(deps.GetSQLDatabase().ReadDB(), deps.GetSQLDatabase().WriteDB())
 		{{if not .MongoDeps}}// {{end}}setSharedRepoMongo(deps.GetMongoDatabase().ReadDB(), deps.GetMongoDatabase().WriteDB())
+		{{if not .ArangoDeps}}// {{end}}setSharedRepoArango(deps.GetArangoDatabase().ReadDB(), deps.GetArangoDatabase().WriteDB())
```

Comment:
berhubung arangonya masuk ke extended, ngambilnya gini aja
```suggestion
		{{if not .ArangoDeps}}// {{end}}setSharedRepoArango(deps.GetExtended("arangodb").(arango.ArangoDatabase).ReadDB(), deps.GetExtended("arangodb").(arango.ArangoDatabase).WriteDB())
```

sama tambahin import klo pake arango
```
{{if .ArangoDeps}}arango "github.com/golangid/candi-plugin/arangodb-adapter"{{end}}
```